src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/EnvelopeProcessor.java

```diff
@@ -117,26 +118,33 @@ public Envelope getEnvelopeByFileAndContainer(String container, String zipFileNa
      * @param envelope to assert against
      * @param pdfs     to assert against
      */
-    public void assertEnvelopeHasPdfs(Envelope envelope, List<Pdf> pdfs) {
+    public static void assertEnvelopeHasPdfs(Envelope envelope, List<Pdf> pdfs) {
```

Comment:
hmmm ... 
Is it really an assertion or more a validation ?
Surely this could be put on the Envelope instead....
Seems the encapsulation is a bit violated here.
maybe something like 
val missingFiles = envelope.getMissingPdfs( pdfs)